portals/devportal/src/main/webapp/source/src/app/components/Apis/Chat/AISearchAssistant.jsx

```diff
@@ -0,0 +1,105 @@
+import React, { useEffect, useState } from 'react';
+import { Box } from '@mui/material';
+import Settings from 'Settings';
+import User from 'AppData/User';
+import Utils from 'AppData/Utils';
+import ChatBotIcon from './ChatIcon';
+import ChatWindow from './ChatWindow';
+
+/**
+ * Renders AI Search Assistant view..
+ * @returns {JSX} renders Chat Icon view.
+ */
+function AISearchAssistant() {
+    const introMessage = {
+        role: 'assistant',
+        // eslint-disable-next-line max-len
+        content: 'Hi there! I\'m Marketplace Assistant. I can help you with finding APIs and providing information related to APIs. How can I help you?',
+    };
+    const [user, setUser] = useState('You');
+
+    const getUser = (environmentName = Utils.getCurrentEnvironment().label) => {
+        const userData = localStorage.getItem(`${User.CONST.LOCALSTORAGE_USER}_${environmentName}`);
+        const partialToken = Utils.getCookie(User.CONST.WSO2_AM_TOKEN_1, environmentName);
+        const refreshToken = Utils.getCookie(User.CONST.WSO2_AM_REFRESH_TOKEN_1, environmentName);
+
+        const isLoginCookie = Utils.getCookie('IS_LOGIN', 'DEFAULT');
+        if (isLoginCookie) {
+            Utils.deleteCookie('IS_LOGIN', Settings.app.context, 'DEFAULT');
+            localStorage.removeItem(`${User.CONST.LOCALSTORAGE_USER}_${environmentName}`);
+            return null;
+        }
+        if (!(userData && (partialToken || refreshToken))) {
+            return null;
+        }
+
+        return User.fromJson(JSON.parse(userData), environmentName);
+    };
+
+    const getMessages = () => {
+        const messagesJSON = localStorage.getItem('messages');
+        const loadedMessages = JSON.parse(messagesJSON);
```

Comment:
No need to assign another variable. Can simply `return JSON.parse(messagesJSON);`